train.py

```diff
@@ -163,7 +171,7 @@ def train(
         default="imle_net",
         help="Select the model to train. (imle_net, mousavi, rajpurkar)",
     )
-    parser.add_argument("--batchsize", type=int, default=32, help="Batch size")
+    parser.add_argument("--batchsize", type=int, default=32, help="Batch size") # Reduce the batch size if any memory related problem comes up
```

Comment:
there is a help tag to provide info for argparser

Reply:
Add the comment into that

Reply:
Done